src/js/_enqueues/lib/comment-reply.js

```diff
@@ -320,6 +322,9 @@ window.addComment = ( function( window ) {
 		cancelElement.style.display = '';
 		addBelowElement.parentNode.insertBefore( respondElement, addBelowElement.nextSibling );
 
+		var replyHeading = getElementById('reply-title');
+		replyHeading.innerHTML = replyHeading.innerHTML.replace(/[a-zA-Z0-9 ]*/, replyTo + ' ')
```

Comment:
CS, also wondering if `innerHTML` is safe both when moving the page and on the comment reply specific page.

```suggestion
		replyHeading.innerHTML = replyHeading.innerHTML.replace( /[a-zA-Z0-9 ]*/, replyTo + ' ' );
```

Reply:
Oooh good point. The added text is coming from `$comment->comment_author `, but if we don't want to assume that's safe, then the best way forward is to enclose the content we want to change in a `<span>` or something, and use `textContent` instead. Would adding extra markup break anything though?